src/brijesh/assignment44/MapFindFrequencyOfNumber.java

```diff
@@ -0,0 +1,30 @@
+package brijesh.assignment44;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.TreeMap;
+
+/*
+Find frequency of each number from the Array
+Integer arr[] = {23,12,45,67,23,12,67,33};
+*/
+
+public class MapFindFrequencyOfNumber {
+
+	public void findFrequencyOfNumber(int[] numberArray) {
+		System.out.println("Given Number Array: "+Arrays.toString(numberArray));
+		Map<Integer,Integer> map = new TreeMap<>();
+		for(Integer key : numberArray) {
+			if(map.containsKey(key))
```

Comment:
2nd approach is
map.put(key, map.getOrDefault(key, 0) + 1);
try once at your end.